genie/input_to_database.py

```diff
@@ -165,6 +166,75 @@ def _get_status_and_error_list(valid, message, entities):
     return input_status_list, invalid_errors_list
 
 
+def get_ancillary_files(
+    syn: synapseclient.Synapse,
+    synid: str,
+    project_id: str,
+    center: str,
+    process: str = "main",
+    downloadFile: bool = True,
+    genie_config: list = None,
+    format_registry: list = None,
+) -> Dict[str, Dict[str, object]]:
+    """Walks through each center's input directory
+    to get a dict of center files
+
+    Args:
+        syn (synapseclient.Synapse): Synapse connection
+        synid (str): Synapse Id of a folder
+        project_id (str): GENIE Synapse project id
+        center (str): GENIE center name
+        process (str, optional): Process type include "main", "mutation".
+                                 Defaults to "main".
+        downloadFile (bool, optional): Downloads the file. Defaults to True.
+
+    Returns:
+        dict: {entity_name: {
+            entity: Synapse.File
+            filetypeformat_object: FileTypeFormat
+            }
+    """
+    logger.info("GETTING {center} INPUT FILES".format(center=center))
+
+    center_files = synapseutils.walk(syn, synid)
+    prepared_center_files = {}
+
+    for _, _, entities in center_files:
+        for name, ent_synid in entities:
+            prepared_center_files[name] = {}
+            # This is to remove vcfs from being validated during main
+            # processing. Often there are too many vcf files, and it is
+            # not necessary for them to be run everytime.
+            if name.endswith(".vcf") and process != "mutation":
+                continue
+
+            ent = syn.get(ent_synid, downloadFile=downloadFile)
+            prepared_center_files[name]["entity"] = ent
+
+            validator = validate.GenieValidationHelper(
+                syn=syn,
+                project_id=project_id,
+                center=center,
+                entitylist=[ent.path],
+                format_registry=format_registry,
+                genie_config=genie_config,
+                ancillary_files=None,
+            )
+            filetype = validator.file_type
```

Comment:
Two paths forward, either handle the custom clinical class, OR... comment this section out and just return a dictionary of entities:
```
{"filename": {"entity": entity}}
```